source/server/spongineer/incompatible.rst

```diff
@@ -14,109 +14,139 @@ To help you resolve which is which, and how to fix the problem (if possible), we
-load the Sponge version of Mixin first, and can resolve many problems in one step.
-
-FarSeek
-~~~~~~~
-- Dependents: Streams mod
-- Versions: 2.3, 2.4
-- Problem: Crash on startup / Failed world generation
-- Solution: Update to version 2.5.
-
-FoamFix
-~~~~~~~
-- Versions: Up to 0.10.7
-- Problem: Crash on startup (block collision error)
-- Solution: Update to version 0.10.8 or newer, or change these two configuration options:
-  ``optimizedBlockPos=false`` and ``patchChunkSerialization=false``
-- *Newer versions of FoamFix (0.10.8+) automatically correct this when SpongeForge is present.*
-
-ForgeEssentials
-~~~~~~~~~~~~~~~
-- Versions: Up to 12.3.82
-- Problem: Crash on startup (mixin NoSuchMethodError)
-- Solution: Update to 12.3.83 or newer.
-- *Newer versions of ForgeEssentials have updated their Mixin usage to version 0.8.*
+SpongeForge jar file name with ``aaa_``, giving a filename like ``aaa_spongeforge-1.16.5-36.2.5-8.1.0-RC1312``. 
+This makes it load the Sponge version of Mixin first, and can resolve many problems in one step.
 
-Hammer Core
-~~~~~~~~~~~
-- Dependents: Many
-- Versions: 2.0.6.1.2+
-- Problem: Crash on startup. Typically, the following error message is returned:
-
-.. code-block:: none
-
-    org.spongepowered.asm.mixin.injection.throwables.InjectionError: Critical injection failure: Redirector onUpdateTileEntities(Lnet/minecraft/util/ITickable;)V in mixins.common.core.json:world.WorldMixin failed injection check, (0/1) succeeded. Scanned 1 target(s). Using refmap mixins.common.refmap.json
-
-- Solution: Change ``<entry key="World.ITickable.Override">true</entry>`` to ``false`` in the ``/asm/hammercore.xml`` file
-
-ICBM Classic
-~~~~~~~~~~~~
-- Versions: All
-- Problem: Missiles stop in midair and appear to lag in place.
-- Solution: Add the commented text below to the ``entity-activation-range/mods`` section of the ``sponge/global.conf`` file:
-
-.. code-block:: none
-
-    entity-activation-range {
-        mods {
-            # COPY THIS - START
-            icbmclassic {
-                defaults {
-                    misc=16
-                }
-                # If 'false', entity activation rules for this mod will be ignored and always tick.
-                enabled=false
-                entities {
-                    missile=16
-                    seat=16
-                }
-            }
-            # COPY THIS - END
-        }
-    }
-
-Just Enough IDs (JEID)
-~~~~~~~~~~~~~~~~~~~~~~
-- Versions: Up to 1.0.3-48
-- Problem: Crash on startup (mixin conflict).
-- Solution: Update to version 1.0.3-54 or newer.
-
-LagGoggles/TickCentral
-~~~~~~~~~~~~~~~~~~~~~~
-- Versions: All
-- Problem: Crash on startup (mixin conflict).
-- Solution: Update to Spongeforge 7.2.0 or newer.
-
-MystCraft
+Forge Permissions
+~~~~~~~~~~~~~~~~~
+
+Some mods may register there own permissions with Forge. These cannot be tracked by mods including SpongeForge due to a 
```

Comment:
there -> their
 commas needed:  `These cannot be tracked by mods, including SpongeForge, due to [...]`

Reply:
Updated